app/src/main/java/org/mifos/mobile/ui/activities/IntroSplashScreen.java

```diff
@@ -0,0 +1,184 @@
+package org.mifos.mobile.ui.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.viewpager.widget.ViewPager;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.TextView;
+
+import com.google.android.material.tabs.TabLayout;
+
+import org.mifos.mobile.R;
+import org.mifos.mobile.utils.IntroViewPagerAdapter;
+import org.mifos.mobile.utils.ScreenItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class IntroSplashScreen extends AppCompatActivity {
+
+    private ViewPager viewPager;
+    IntroViewPagerAdapter introViewPagerAdapter;
+    TabLayout tabIndicator;
+    TextView btnNext;
+    int position = 0;
+    TextView btnGetStarted;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // make the activity on full screen
+//        requestWindowFeature(Window.FEATURE_NO_TITLE);
+//        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
+//                WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+        setContentView(R.layout.activity_intro_splash_screen);
+
+
+
+//        if (restorePrefData()) {
+//
+//            Intent mainActivity = new Intent(getApplicationContext(), LoginActivity.class);
+//            startActivity(mainActivity);
+//            finish();
+//        }
+
+        // hide the action bar
+
+        //getSupportActionBar().hide();
+
+        // ini views
+        btnNext = findViewById(R.id.btnNext);
+        btnGetStarted = findViewById(R.id.getStarted);
+        tabIndicator = findViewById(R.id.tabLayout);
+//        btnAnim = AnimationUtils.loadAnimation(getApplicationContext(),R.anim.button_animation);
+//        tvSkip = findViewById(R.id.tv_skip);
+
+        final List<ScreenItem> mList = new ArrayList<>();
+        mList.add(new ScreenItem("Bank account management", "With a bank account management feature, users can monitor their cards and bank account.", R.drawable.management));
+        mList.add(new ScreenItem("Tracking spending habits", "Users can get more control over their finances and reach their saving goals." , R.drawable.track_money));
+        mList.add(new ScreenItem("Customer Support","Customer can get professional consultation or advice anytime they need it.", R.drawable.customer_support));
```

Comment:
Here also more than 100 words issue and also here at 53 characters "," is not followed by white space.